website/docs/cdktf/concepts/variables-and-outputs.mdx

```diff
@@ -100,6 +100,14 @@ new TerraformResource(this, "resource", {
 });
 ```
 
+### Passing Input Variables to CDKTF
+
+You can pass input variables e.g. with the name `imageId` to CDKTF in the following ways:
```

Comment:
```suggestion
You can pass input variables to `cdktf diff`, `cdktf deploy`, and `cdktf destroy` in the following ways:
```

Reply:
I think the name of the variable (`imageId`) is something that can be inferred – it can also be found in the example above.